docs/source/examples.rst

```diff
@@ -3,19 +3,64 @@ Examples
 
 .. currentmodule:: examples
 
-Vision
+In this section, you will find the data loading implementations (using DataPipes) of various
+popular datasets across different research domains.
+
+Audio
 -----------
 
+LibriSpeech
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+`LibriSpeech dataset <https://www.openslr.org/12/>`_ is corpus of approximately 1000 hours of 16kHz read
+English speech. Here is the
+`DataPipe implementation of LibriSpeech <https://github.com/pytorch/data/blob/main/examples/audio/librispeech.py>`_
+to load the data   .
+
 Text
 -----------
 
-Audio
+CC-100
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+`CC-100 : Monolingual Datasets from Web Crawl Data <http://data.statmt.org/cc-100/>`_ is a corpus comprises of
+monolingual data for 100+ languages and also includes data for romanized languages. You can find the DataPipe
+implementation in `this Jupyter Notebook <https://github.com/pytorch/data/blob/main/examples/text/CC100.ipynb>`_.
+
+
+IMDB
```

Comment:
And, all TorchText Datasets are fully revamped in TorchText repo here: https://github.com/pytorch/text/tree/main/torchtext/datasets